app/assets/locales/android_translatable_strings.txt

```diff
@@ -116,11 +117,16 @@ sync.fail.bad.data=Server provided improperly formatted data, please try again o
 sync.fail.bad.local=Your information was fetched, but a problem occurred with the log in. Please try again.
 sync.fail.bad.network=Couldn't contact server. Please make sure an internet connection is available or try again later.
 sync.fail.server.error=The server encountered an error processing your data. Please try again later. If the problem persists, contact CommCare Support.
-sync.fail.rate.limited.server.error=Our servers are unavailable at this time. Please try again later.
+sync.fail.rate.limited.server.error=Form submission rate limit reached. Your forms are being saved locally and will be submitted to the server later.
 sync.fail.unknown=An unexpected issue occurred during sync. Please try to sync again.
 sync.fail.unsent=Having issues communicating with the server to send forms. Will try again later.
 sync.fail.individual=1 or more forms failed to send due to individual problems. See quarantined forms for details.
 
+form.send.rate.limit.error.title=You've reached your form submission rate limit
+form.send.rate.limit.error.message=Form submissions will be saved locally on your phone until rate limiting expires. Not to worry, all forms will be auto-synced to the server later. Contact your CommCare admin for more information.
+dialog.button.do.not.show.again=Don’t show again

```

Comment:
can we prefix they keys like rate.limit.error.diaglog..... just to make them more specific.